### DISCUSSION TITLE
اجرا نشدن

### DISCUSSION THREAD

**Discussion by m0p0m** [hiddify/hiddify-app#610] [General]
سلام من پروژه رو اجرا میکنم ولی با مشکل مواجه میشم توی فایل yaml از نسخه 3.2.0 به بالا استفادع شده در حالی که اخرین نسخه فلاتر 3.19.2 هستش در نتیجهیه سری فایل ها هم وجود نداشتن داخل سورس چه کاری باید انجام بدم تا اجرا بشه؟

**Comment by LAziznezhad** [comment]
منم نتونستم اجرا کنم ارور داره

**Reply by m0p0m** [reply]
به دلیل تغیرات تو کتابخونه هاشه که نمیشه

**Reply by LAziznezhad** [reply]
درستش کردم اوکی شد

**Comment by LAziznezhad** [comment]
میتونم ایدی تلگرامتون رو داشته باشم چون نتوستم من اجرا کنم یکم رهنمای کنید ممنون میشم
